### ISSUE TITLE
Subtitle Should be in a Header Block in <p> tags instead of a <h2>

### ISSUE THREAD

**Issue by carolyncole** [psu-libraries/scholarsphere-3#1174]
Check out the right way to do a subtitle discovered by @mtribone 
http://w3c.github.io/html/common-idioms-without-dedicated-elements.html#common-idioms-without-dedicated-elements
